### ISSUE TITLE
Skillet (unable to "link recipe" and missing "Earn some money!" 

### ISSUE THREAD

**Issue by Megiia** [b-morgan/Skillet-Classic#193]
Hey there! 

So right-clicking and then "link recipe" doesn't do anything for me. I tried party, whispers, raid, chat open, chat closed, removed all other addons, and it still doesn't work. 
As for the "earn some money" icon, its completely missing. 
Now heres the catch... if I am in combat BOTH work!  Please help me. I attached a small video how the interface looks before and after combat,. 

https://github.com/user-attachments/assets/bc8535d0-f627-464f-97d7-d87e8ca35be5

**Comment by b-morgan** [comment]
Right-click and selecting Link Recipe doesn't work because Classic Era (and its derivatives) do not have recipe links (i.e. GetTradeSkillRecipeLink returns nil).

Skillet-Classic doesn't have an "earn some money" icon and doesn't work very well in combat.

When you are in combat, you are using a different addon (which I don't recognize).

Skillet-Classic 2.05 (in alpha now, soon to be released) will remove Link Recipe from the right-click menu.

Skillet-Classic does have a "Send Reagents" which is triggered by an Alt-click on the item icon in the detail (right) side of the frame. I'll look into adding that to the right-click menu on the left side as a replacement for Link Recipe.

**Comment by b-morgan** [comment]
Skillet-Classic-2.05-alpha4 adds "List Reagents" to the right-click menu on a recipe. This is the same output as alt-click on the item icon. For the recursive output, you will still need to ctrl-alt-click the detail icon.
